Source/Shared/arcana/functional/inplace_function.h

```diff
@@ -33,24 +33,38 @@
 namespace stdext
 {
     constexpr size_t InplaceFunctionDefaultCapacity = 32;
-
-    enum class inplace_function_operation
+        
+    template<bool SupportsCopy>
+    struct inplace_function_operation
```

Comment:
This isn't your code, but I'm not sure why this enum class is outside of the template class? It seems to be a private implementation detail. Is it not possible to put inside the inplace_function class?

Reply:
I actually tried to move this inside and thereby discovered why it's outside. These operations can be used in calls between different templated instances of `inplace_function` -- for example, between `inplace_function`s of differing allocation or buffer size -- which the type system will prevent if they're internal to the templated classes. We could `underlying_cast` or try to do something a little tricky hide this a bit better, but after realizing why it was where it was, I felt like going down that road probably wasn't a great fit for this PR, at least.